Step07/app/src/main/java/net/agilepartner/workshops/cqrs/core/Event.java

```diff
@@ -1,10 +1,17 @@
 package net.agilepartner.workshops.cqrs.core;
 
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
 import java.util.UUID;
 
+@Getter
+@Setter
 public abstract class Event implements Message {
-    public UUID aggregateId;
```

Comment:
I understand it seems counter-intuitive to have public fields, but most likely the events will be serialized, and there is no real need for encapsulation here, since events (and command for that matter) are mainly data containers, and have no logic. 
Therefore, I go for simplicity and don't need getters and setters that just add syntactic noise here... using the _black magic_ of `lombok`

Reply:
Sure, it's counter-intuitive. And in case field are not encapsulated, the temptation is use something like lombok.